strapi/strapi
# fix: replace all escaped `\n` with an actual `\n` character

### What does it do?

When our objects are stringified they're parsed so the `\n` becomes `\\n`, which means when it's parsed, it's no longer recognised as a line-break character, instead it's recognised as a `\` and `n` separately. This PR replaces those string sections, with the line-break character again.

### Why is it needed?

IMO it's a little confusing because `\n` isn't strictly markdown and if people use `<br>` instead, everything works fine. But this solution does solve the issue brought up by the community.

### How to test it?

* Add richtext content field to a content type
* Set a default value of something like `hello\nthere`
* Create new content entry based on initial type
* Select preview mode
* There should be a line break.

This will also work if you type out line breaks in the editor itself.

### Related issue(s)/PR(s)

* resolves #13093
